app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= provide :title, "Home" %>
+
+<h1>Sample App</h1>
+  <p>
+    This is the home page for the
+    <%= link_to "Ruby on Rails Tutorial", Settings.links.rails_tutorial.home_page %>
+    sample application.
+  </p>
```

Comment:
remove 2 spaces

Reply:
done but can you check again if I do it correctly or not